docs/source/services/qos_service/enabling-rate-limiting.rst

```diff
@@ -0,0 +1,42 @@
+.. _enabling-rate-limiting:
+
+Enabling Rate Limiting
+======================
+
+Currently the QoS service allows a client to configure the read and write limits and enforces these limits.
+Detailed documentation is :ref:`here <qos-client-configuration>`.
+
+QoS also publishes client level metrics for the following:
+
+        - Meter readRequestCount
```

Comment:
Specify the common prefix (com.palantir...) and say that they are all meters instead of repeating it every time